docs/getting-started/built-ins.md

```diff
@@ -105,7 +105,32 @@ const s1 = toByteString('793ff39de7e1dce2d853e24256099d25fa1b1598ee24069f24511d7
 reverseByteString(s1, 32) // 6cfeea2d7a1d51249f0624ee98151bfa259d095642e253d8e2dce1e79df33f79
 ```
 
-### Bitshift
+### Bitwise Operator
+
+- `and(x: bigint, y: bigint): bigint` Bitwise AND
+
+```typescript
+and(13n, 5n) // 5n
```

Comment:
Add more complex examples, like, 32/64, or even 256 bits, especially when two operands are of different length.